src/geom/reference_elem.data

```diff
@@ -6,6 +6,20 @@
   typedef std::map<ElemType, std::string> FileMapType;
   FileMapType ref_elem_file;
 
+static const char* one_node =
```

Comment:
I think it would be better to call this string `one_nodeelem` since it refers to a NodeElem and not a Node?

Reply:
done